configure.ac

```diff
@@ -868,6 +868,21 @@ case "$host_os" in
 
     AC_DEFINE(OPEN_BSD,,OpenBSD)
     AC_DEFINE(KAME,1,KAME IPv6)
+
+    if test "x${enable_pimd}" != "xno"; then
+      BSDVER=`uname -r`
```

Comment:
I'd do this as
```
if test "x${enable_pimd}" != "xno"; then
  case "$host_os" in
  openbsd6.0)
    ;;
  openbsd[6-9]*)
    AC_MSG_FAILURE([pimd cannot be enabled as PIM support has been removed from OpenBSD 6.1])
    ;;
  esac
fi
```

There are 3 distinct sub-points here:
* checking $host_os instead of uname means checking the compilation target instead of the build host. Not sure if it's possible to cross-compile for OpenBSD, but if it is, uname would be the wrong check on crosscompile.
* the case easily allows matching a specific range of OpenBSD versions
  * i don't think it'll be re-added that quickly, and i'd prefer giving the user an useful error message...
* configure should fail if the user specifies `--enable-pimd`. This is "common autoconf policy"; especially if you have e.g. a buildfarm and are moving from 6.0 to 6.1 you won't see the error message and then be confused why there is no pimd. If a user explicitly gives a configure option, configure should fail if it can't be honored.